cli/root.go

```diff
@@ -96,20 +191,63 @@ func init() {
 	})
 }
 
+func initEndureConfig() *EndureConfig {
+	c := &config.Viper{
+		Path:   CfgFile,
+		Prefix: "rr",
+	}
+	err := c.Init()
+	if err != nil {
+		panic(err)
+	}
+
+	// init default config
+	if !c.Has(EndureKey) {
+		return &EndureConfig{
+			GracePeriod: time.Second * 30,
```

Comment:
Too small default graceful period. Ideally, it should be unlimited or large enough to cover most jobs.

Reply:
This is the standard grace period used in all container systems such as Kubernetes or docker. It can't be unlimited, for uses who need a larger period, there is an ability to set it in hours. For example `999999h` -> almost 10 years.